.changeset/silent-otters-accept.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/account": patch
+"fuels": patch
+---
+
+fix: ensure that we fetch node info per estimation
```

Comment:
Can we go with a more concise release note in the PR desription?

 - Improved `Provider` cache to self-clean on TX dry-run/estimation

Reply:
Good idea